packages/suite/src/reducers/store.ts

```diff
@@ -100,13 +108,20 @@ export const initStore = (
 
     const patchedState =
         preloadedState && statePatch && patchConfirm(statePatch)
-            ? mergeDeepObject.withOptions({ dotNotation: true }, preloadedState, statePatch)
-            : preloadedState;
+            ? (mergeDeepObject.withOptions(
+                  { dotNotation: true },
+                  preloadedState,
+                  statePatch,
+              ) as unknown as AppState)
```

Comment:
`as unknown as AppState` just screams bad pattern. Although in this case it is ok, because `statePatch` is an external input for advanced users who want to pass custom object through process param.
Still, I think this would be slightly smoother :slightly_smiling_face:  
```suggestion
                  statePatch as Partial<AppState>,
              ))
```